Xamarin.PropertyEditing/Reflection/ReflectionEnumPropertyInfo.cs

```diff
@@ -65,7 +65,7 @@ public override void SetValue<TValue> (object target, TValue value)
 			}
 		}
 
-		public override TValue GetValue<TValue> (object target)
+		public override async Task<TValue> GetValueAsync<TValue> (object target)
```

Comment:
And to clarify, it's completely ok for these to run synchronously, regardless of whether it's supposed to be a good implementation or not. We're more enabling providers (like yours) to be async if they need to be, not necessarily saying they _should_ be.